.github/workflows/backend-integration-test.yml

```diff
@@ -76,8 +76,8 @@ jobs:
         uses: docker/setup-qemu-action@v2
 
       - name: Package Gravitino
+        if : ${{ matrix.test-mode == 'deploy' }}
         run: |
-          ./gradlew build -x test -PjdkVersion=${{ matrix.java-version }}
```

Comment:
Can we remove this step, I remembered that the `compileDistribition` will not build the trino module, this will lead to the test failure, can you please confirm this?

Reply:
<img width="1135" alt="image" src="https://github.com/datastrato/gravitino/assets/15794564/f516eeb2-1f1f-4166-9dfb-b56667081e5b">

Task `assembleDistribution` will build the Trino module by the way.